modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portal/ServerStatistics.java

```diff
@@ -24,12 +24,12 @@
  * @author Michael C. Han
  * @author Brian Wing Shun Chan
  */
-@Component(
-	enabled = false, property = "namespace=com.liferay.monitoring.Portal",
-	service = {DataSampleProcessor.class, ServerStatistics.class}
-)
-public class ServerStatistics
-	implements DataSampleProcessor<PortalRequestDataSample> {
+@Component(service = ServerStatistics.class)
+public class ServerStatistics {
+
```

Comment:
Hey @RenanUmV, now this class looks like a helper class. Could you please rename this to reflect this change?

Thx